code/src/components/Summary.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import Bottles from '../image/Bottles.png'
+
+export const Summary = ({ name, wineColor, country }) => {
```

Comment:
when you only have one component in a file, you could use export default instead. It would be a clear sign to others who read the code that there is only one compoment and that all of it is exported.